index.html

```diff
@@ -0,0 +1,135 @@
+      </a>
+    </div>
+    <div class="bars" id="toggle-icon" onclick="toggleMobileMenu(this)">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+      <ul class="nav-menu">
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a>Program</a>
+        </li>
+        <li>
+          <a>Join</a>
+        </li>
+        <li>
+          <a>Sponsor</a>
+        </li>
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="links-container">
+    <ul>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a>Program</a>
+      </li>
+      <li>
+        <a>Join</a>
+      </li>
+      <li>
+        <a>Sponsor</a>
+      </li>
+      <li>
+        <a>News</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+
+<header class="header">
+  <div class="header-container">
+    <h1 class="welcome-heading">"Hello! Sharing world"</h1>
+    <h2 class="intro">10-Gano Courses</h2>
+    <p class="intro-paragraph">Learn at the comfort of your own home. 10-Gano website is the world's best platform for whoever is willling to develop their skillset and advance career.</p>
+    <!-- today's date -->
+    <h3 class="date" id="date"></h3>
+  </div>
+</header>
+
+<section class="main-program">
+  <h3 class="main-program-heading">Main Program</h3>
+  <hr class="main-program-hr">
+  <ul class="programs-list">
+    <li class="program-container">
+      <img src="assets/distance-learning.png" alt="course" />
+      <h4 class="program-title">AI Marketing</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/online-test.png" alt="course" />
+      <h4 class="program-title">Fintech Programme</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/audio-book.png" alt="course" />
+      <h4 class="program-title">Cyber Security</h4>
+      <p class="program-desc">Try creating your own work using open source license</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/student.png" alt="course" />
+      <h4 class="program-title">AI Marketing</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/work.png" alt="course" />
+      <h4 class="program-title">Blockchain Strategy Programme</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+  </ul>
+  <button type="button" class="program-join-button">Join CC Latest Course Journal</button>
```

Comment:
- [ ] KIndly make sure the `Join CC Latest Course Journal` button is visible only on the mobile version and make sure on the desktop to add the `SEE THE WHOLE PROGRAM` in order to match the provided Behance design as much as possible.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225590960-c6f05f3d-91c1-4164-99e9-8f9095e99e1b.png)

   - ### BEHANCE DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225591025-db8ff933-9485-420c-a666-637722ee0102.png)